.travis.yml

```diff
@@ -38,8 +42,15 @@ branches:
 
 install:
   - if [[ $PHPUNIT = "local" ]]; then composer install; fi
+  - if [[ $GRUNT = "yes" && "$TRAVIS_PULL_REQUEST" != "false" ]]; then chmod +x bin/init-grunt.sh; . bin/init-grunt.sh; fi
 
 before_script:
   - bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
 
-script: if [[ $PHPUNIT = "local" ]]; then ./vendor/bin/phpunit; else phpunit; fi
+script: 
+  - if [[ $GRUNT = "yes" && "$TRAVIS_PULL_REQUEST" != "false" ]]; then grunt; fi
+  - if [[ $PHPUNIT = "local" ]]; then ./vendor/bin/phpunit; else phpunit; fi
+
+after_failure:
+- cat "logs/phpcs.log"
+- cat "logs/jslogs.log"
```

Comment:
I believe this would need a new line at the end of file.

Reply:
done